packages/contracts-bedrock/package.json

```diff
@@ -19,7 +19,7 @@
     "build:differential": "tsc scripts/differential-testing.ts --outDir dist --moduleResolution node --esModuleInterop",
     "prebuild": "yarn ts-node scripts/verify-foundry-install.ts",
     "build": "hardhat compile && yarn autogen:artifacts && yarn build:ts && yarn typechain",
-    "build:ts": "tsc -p tsconfig.json",
+    "build:ts": "tsc -p tsconfig.build.json",
```

Comment:
Why build.json here?

Reply:
If we don't include the scripts/deploy-config/tasks folders in tsconfig then vscode shows errors and it also will not automatically lint your files. However, if you do include those files then they will be compiled and included in the final build. The solution is to use two different files, one that gets used during development and includes all of your folders and one that gets used during the build and only included your source folder.